UPGRADE-3.x.md

```diff
@@ -23,9 +23,9 @@ Instead of a simple boolean var (whether to persist or not filters) you can now
 that will be responsible for doing the job (see `FilterPersisterInterface`).
 An implementation was added, which falls back to the previous behavior : `SessionFilterPersister`.
 
-## Deprecated editing of a child admin that does not belong to a given parent
+## Deprecated edit/show/delete of a child admin that does not belong to a given parent
 
-This is not allowed anymore and will throw a 404 error.
+This is not allowed anymore and will throw a 404 error in future.
```

Comment:
"in the future"

Reply:
One day I will write everything correctly :sweat_smile: